strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java

```diff
@@ -0,0 +1,258 @@
+package org.carlspring.strongbox.controllers.users;
+
+import org.carlspring.strongbox.controllers.BaseController;
+import org.carlspring.strongbox.users.domain.AccessModel;
+import org.carlspring.strongbox.users.domain.User;
+import org.carlspring.strongbox.users.service.UserService;
+import org.carlspring.strongbox.users.userdetails.SpringSecurityUser;
+
+import javax.inject.Inject;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+import io.swagger.annotations.*;
+import org.apache.commons.lang3.StringUtils;
+import org.jose4j.lang.JoseException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/users")
+@Api(value = "/users")
+public class UserController
+        extends BaseController
+{
+
+    @Inject
+    UserService userService;
+
+    // ----------------------------------------------------------------------------------------------------------------
+    // This method exists for testing purpose
+
```

Comment:
Cosmetic: The annotations across this class seems to be a bit messed up. Would you mind re-indenting them according to the convention?